dropwizard-benchmarks/pom.xml

```
@@ -21,6 +21,8 @@
         <maven.site.skip>true</maven.site.skip>
         <maven.site.deploy.skip>true</maven.site.deploy.skip>
         <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+        <!-- Ignore test coverage metrics for this module  -->
+        <sonar.coverage.exclusions>**/*.java</sonar.coverage.exclusions>
```

@rhowe Thanks for your contribution!

I think setting `sonar.skip` to `true` should do the trick: https://docs.sonarqube.org/9.3/analysis/scan/sonarscanner-for-maven/#header-6

```suggestion
        <sonar.skip>true</sonar.skip>
```

---

I found that as well, but that will skip all Sonar analysis including code smells etc which seems a _bit_ strong?

---

@rhowe Ah, I see. You really only wanted to disable coverage metrics. Nevermind then… 😅

---

Hm, I like that approach. Are you able to trigger a sonar run against this branch somehow to see if the change actually does the right thing?